qa/tasks/cephfs/test_volumes.py

```diff
@@ -1817,8 +1837,7 @@ def test_subvolume_snapshot_clone_and_reclone(self):
         # remove snapshot
         self._fs_cmd("subvolume", "snapshot", "rm", self.volname, subvolume, snapshot)
 
-        # verify clone
-        self._verify_clone(subvolume, clone1)
+        # NOTE: cannot verify clone as source snapshot is removed
```

Comment:
nit: should we move verify_clone before removing snapshot in all the places where possible and remove this note as it's obvious from now?

Reply:
Fair enough, will do the same (add `verify_clone` and remove the snapshot post the same).